js/main.js

```diff
@@ -229,11 +229,16 @@ $(window).bind('keydown', function(e) {
         // Select all text in address bar
         $('.js-navAddressBar').select();
         break;
+      case config.keyShortcuts.save:
+        // Trigger any visible .saveForm button
+        // .last() will make sure you'll only click one button if 2 are visible (listing form)
+        $('.js-saveForm:visible').last().trigger('click');
```

Comment:
Why do you think a listener is more complex than code outside the view triggering actions on it? 

And why would depending on the layout being in a certain state be better than having each view trigger it's save action on its own?